EKF/ekf_helper.cpp

```diff
@@ -1755,7 +1762,7 @@ bool Ekf::resetYawToEKFGSF()
 
 void Ekf::requestEmergencyNavReset()
```

Comment:
Is the purpose of this function to give the possibility to the client to trigger a yaw reset using the solution of he yaw estimator? It's never used at the moment.

Reply:
Yes, the original experimentation tried to get the commander failsafe to trigger it prior to entering a non-position mode, but I postponed that at the time as it was getting too complicated to try and get both changes in. I would still like to investigate the possibility of doing so. I don't mind removing that function if flash space is an issue.